mdb-realm/src/helpers.ts

```diff
@@ -178,91 +90,93 @@ export async function waitForClusterDeployment(config: EnvironmentConfig): Promi
     throw new Error(`Cluster failed to deploy after ${100 * pollDelay} seconds`);
 }
 
-export async function configureRealmCli(config: EnvironmentConfig): Promise<void> {
-    await execCmd("npm i -g mongodb-realm-cli");
-
-    await execCliCmd(
-        `login --api-key ${config.apiKey} --private-api-key ${config.privateApiKey} --atlas-url ${config.atlasUrl} --realm-url ${config.realmUrl}`,
-    );
-}
-
-export async function publishApplication(appPath: string, config: EnvironmentConfig): Promise<{ id: string }> {
-    const appName = `${path.basename(appPath)}-${getSuffix()}`;
-    core.info(`Creating app ${appName}`);
-
-    const createResponse = await execCliCmd(`apps create --name ${appName}`);
-
-    const appId = createResponse.map(r => r.doc).find(d => d && d.client_app_id).client_app_id;
-
-    core.info(`Created app ${appName} with Id: ${appId}`);
+export async function deleteApplications(config: EnvironmentConfig, deleteAll = false): Promise<void> {
```

Comment:
I think this would be more consistent across the code.

```suggestion
export async function deleteApps(config: EnvironmentConfig, deleteAll = false): Promise<void> {
```